docker/P2P-image/Dockerfile

```diff
@@ -63,7 +63,8 @@ RUN grep -v 'black' install/requirements.txt | xargs pip3 install  --no-cache-di
     && npm install
 
 # set use_p2p to yes in the config file
-RUN sed -i 's/use_p2p = no/use_p2p = yes/g' config/slips.conf
+# RUN sed -i 's/use_p2p = no/use_p2p = yes/g' config/slips.conf
+RUN sed -i 's/use_p2p = no/use_p2p = yes/g' config/slips.yaml
```

Comment:
the purpose of this line in both dockerfiles was to set `use_p2p` to yes instead of no in the config file, now the format of the config file has changes, so we should change the sed cmd as well to use the new format.